src/Helpers/Model.php

```diff
@@ -101,7 +106,11 @@ public function filterContains(): Model
                         $hasColumn = in_array($field, $columnList, true);
 
                         if ($hasColumn) {
-                            $query->orWhere($table . '.' . $field, SqlSupport::like($query), '%' . $this->powerGridComponent->search . '%');
+                            if (DB::getSchemaBuilder()->getColumnType($table, $field) == 'json' && $query->getModel()->getConnection()->getDriverName() != 'pgsql') {
+                                $query->orWhereRaw('LOWER(`' . $table . '` .`' . $field . '`)' . SqlSupport::like($query) . '?', '%' . strtolower($this->powerGridComponent->search) . '%');
```

Comment:
```php
$query->orWhereRaw('LOWER(`' . $table . '` .`' . $field . '`) ' . SqlSupport::like($query) . '?', '%' . strtolower($this->powerGridComponent->search) . '%');
```

I think if adding space before SqlSupport might work

https://w3resource.com/PostgreSQL/lower-function.php

Reply:
I think Postgree doesn't recognize the dot(.) this: LOWER(table.field)